cmd/edit/cluster/cmd.go

```diff
@@ -615,6 +789,53 @@ func run(cmd *cobra.Command, _ []string) {
 		clusterConfig.AdditionalAllowedPrincipals = additionalAllowedPrincipals
 	}
 
+	if allowedRegistries != nil {
+		clusterConfig.AllowedRegistries = allowedRegistries
+	}
+	if blockedRegistries != nil {
+		clusterConfig.BlockedRegistries = blockedRegistries
+	}
+	if insecureRegistries != nil {
+		clusterConfig.InsecureRegistries = insecureRegistries
+	}
+	if allowedRegistriesForImport != "" {
+		obj := map[string]bool{}
+		list := helper.HandleEmptyStringOnSlice(strings.Split(allowedRegistriesForImport, ","))
+		for _, registry := range list {
+			registryObj := helper.HandleEmptyStringOnSlice(strings.Split(registry, ":"))
```

Comment:
we can have a registry with a port and it will have a ":" inside, do you have a test on this?
e.g. "registry.io:80:insecure" works correctly?